src/main/fc/fc_msp.c

```diff
@@ -135,7 +135,7 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT + 1] = {
     { BOXGOV, "GOVERNOR", 18 },
     { BOXOSD, "OSD SW", 19 },
     { BOXTELEMETRY, "TELEMETRY", 20 },
-    { BOXGTUNE, "GTUNE", 21 },
+    { BOXDYNAMICFILTER, "DYNAMIC FILTER", 21 },
```

Comment:
Somehow I thought 31 was the limit and I  couldn't add another mode. Changed this now.